oddsgym/envs/soccer.py

```diff
@@ -91,7 +91,7 @@ def render(self, mode='human'):  # pragma: no cover
         teams_str = ', '.join(['Home Team {} VS Away Team {}'.format(row.home_team, row.away_team)
                                for row in self.teams.iloc[index].itertuples()])
         balance_str = super().render(mode)
-        return '. '.join((teams_str, balance_str))
+        print('. '.join((teams_str, balance_str)))
```

Comment:
Same like previous comment. Also, why is there a different implementation for the Percentage env and the regular env? (why does the Percentage env even implements a render method, it should have been the same one like the regular ThreeWaySoccer env)